R/proportional_hazards-glmnet.R

```diff
@@ -326,29 +323,104 @@ multi_predict._coxnet <- function(object,
     }
   }
 
-  if (type == "linear_pred") {
-    pred <- multi_predict_coxnet_linear_pred(
-      object,
-      new_data = new_data,
-      opts = dots,
-      penalty = penalty
-    )
-  } else {
-    pred <- predict(
-      object,
-      new_data = new_data,
-      type = type,
-      ...,
-      penalty = penalty,
-      multi = TRUE
```

Comment:
This `multi` made it through the glmnet-specific `predict()` methods but can't get through `predict.model_fit()`.